pkg/rest/coverage/fixtures/audit.log

```diff
@@ -0,0 +1,7 @@
+{"kind":"Event","apiVersion":"audit.k8s.io/v1beta1","metadata":{"creationTimestamp":"2019-06-03T12:38:55Z"},"level":"Request","timestamp":"2019-06-03T12:38:55Z","auditID":"test-id-1","stage":"RequestReceived","requestURI":"/pets","verb":"list","objectRef":{},"requestReceivedTimestamp":"2019-06-03T12:38:55.352016Z","stageTimestamp":"2019-06-03T12:38:55.352016Z"}
```

Comment:
It is possible to do that, but I prefer to keep this in a separate file. In case of format change, I would add another file with a new format version (and test old one as well).

Reply:
:+1: makes sense